local_planner/test/test_common.cpp

```diff
@@ -124,103 +124,60 @@ TEST(Common, elevationAnglefromCartesian) {
   EXPECT_FLOAT_EQ(-50.194428, angle_non_zero_origin);
 }
 
-TEST(Common, elevationAngletoIndex) {
-  // GIVEN: the elevation angle of a point and the histogram resolution
-  PolarPoint p_pol_1 = {};
-  p_pol_1.e = 0.f;
-  PolarPoint p_pol_2 = {};
-  p_pol_2.e = 34.f;
-  PolarPoint p_pol_3 = {};
-  p_pol_3.e = 90.f;
-  PolarPoint p_pol_4 = {};
-  p_pol_4.e = -90.f;
-  const float resolution_1 = 3.f;
-  const float resolution_2 = 12.f;
-  PolarPoint p_pol_invalid_1 = {};
-  p_pol_invalid_1.e = 94.f;
-  PolarPoint p_pol_invalid_2 = {};
-  p_pol_invalid_2.e = -999.f;
-  const float resolution_invalid_1 = 0.f;
-  const float resolution_invalid_2 = -1.f;
-
-  // WHEN: we convert the elevation angle to a histogram index
-  const int index_1 = polarToHistogramIndex(p_pol_1, resolution_1).y();
-  const int index_2 = polarToHistogramIndex(p_pol_2, resolution_1).y();
-  const int index_3 = polarToHistogramIndex(p_pol_1, resolution_2).y();
-  const int index_4 = polarToHistogramIndex(p_pol_2, resolution_2).y();
-  const int index_5 = polarToHistogramIndex(p_pol_3, resolution_2).y();
-  const int index_6 = polarToHistogramIndex(p_pol_4, resolution_2).y();
-  const int index_invalid_1 =
-      polarToHistogramIndex(p_pol_invalid_1, resolution_1).y();
-  const int index_invalid_2 =
-      polarToHistogramIndex(p_pol_invalid_2, resolution_1).y();
-  const int index_invalid_3 =
-      polarToHistogramIndex(p_pol_1, resolution_invalid_1).y();
-  const int index_invalid_4 =
-      polarToHistogramIndex(p_pol_1, resolution_invalid_2).y();
-
-  // THEN: the vertical histogram index should be ..
-  EXPECT_EQ(30, index_1);
-  EXPECT_EQ(41, index_2);
-  EXPECT_EQ(7, index_3);
-  EXPECT_EQ(10, index_4);
-  EXPECT_EQ(14, index_5);
-  EXPECT_EQ(0, index_6);
-  EXPECT_EQ(0, index_invalid_1);
-  EXPECT_EQ(0, index_invalid_2);
-  EXPECT_EQ(0, index_invalid_3);
-  EXPECT_EQ(0, index_invalid_4);
-}
-TEST(Common, azimuthAngletoIndex) {
-  // GIVEN: the azimuth angle of a point and the histogram resolution
+TEST(Common, polarToHistogramIndex) {
+  // GIVEN: the polar point and the histogram resolution
   PolarPoint p_pol_1 = {};
   PolarPoint p_pol_2 = {};
   PolarPoint p_pol_3 = {};
   PolarPoint p_pol_4 = {};
   PolarPoint p_pol_5 = {};
-  PolarPoint p_pol_invalid_1 = {};
-  PolarPoint p_pol_invalid_2 = {};
+  PolarPoint p_pol_6 = {};
+  p_pol_1.e = 0.f;
```

Comment:
Use the constructor directly here, will look much nicer. `PolarPoint p_pol_1(34.f, 0.f);` etc...